src/firebolt_db/firebolt_dialect.py

```diff
@@ -111,9 +111,15 @@ def create_connect_args(self, url: URL) -> Tuple[List, Dict]:
         # parameters are all passed as a string, we need to convert
         # bool flag to boolean for SDK compatibility
         token_cache_flag = bool(strtobool(parameters.pop("use_token_cache", "True")))
+        service_account_flag = bool(strtobool(parameters.pop("service_account", "0")))
```

Comment:
""Can we change `"0"` to `"False"` to be consistent?

Reply:
I don't mind either way, I used `"0"` to be aligned with the design doc. Should we keep this consistent across connectors?

Reply:
Set it to False since there's no analogue in other connectors.